_sass/pages/_blog.scss

```diff
@@ -0,0 +1,7 @@
+// Unique elements for blog pages
+
+.blog-page {
+  table.big-query {
+    @include tables(200px);
```

Comment:
I don't think it makes sense to support the "BigQuery" table on blog pages because I don't anticipate a use for that.